packages/api/src/index.ts

```diff
@@ -37,3 +38,14 @@ void (async () => {
   .catch((err) => {
     logger.crit('An error happened during app start', err);
   });
+
+
+  app.get('/login/github', (req, res) => {
+    const url = `https://github.com/login/oauth/authorize?client_id=${CLIENT_ID}&redirect_uri=http://localhost:3000/api/auth/callback/github?&scope=user:email`;
```

Comment:
We'll need to modify this to allow for a dynamic redirect (not just a static `localhost:3000`

We should probably have an env variable for `APP_URL` or something and then use the presence of that and/or the state of `NODE_ENV` to figure out what to do here.

@johnkahn any suggestions?

Reply:
~Yes, and I thought we talked about using Passport.js for this 🤔 Or were we going to refactor this to use Passport.js later?~

Never mind, I remember now. Yeah I wouldn't actually modify that for now. I would leave it as `localhost:3000` for now and we'll modify it when passport is setup.